src/components/selectable/selectable_list/selectable_list.tsx

```diff
@@ -287,6 +287,18 @@ export class EuiSelectableList<T> extends Component<
         ...this.calculateAriaSetAttrs(optionArray),
       });
     }
+
+    // ensure that ListRow updates based on item props
+    if (
+      isVirtualized &&
+      (prevProps.showIcons !== this.props.showIcons ||
+        prevProps.paddingSize !== this.props.paddingSize ||
+        prevProps.textWrap !== this.props.textWrap ||
+        prevProps.onFocusBadge !== this.props.onFocusBadge ||
+        prevProps.searchable !== this.props.searchable)
+    ) {
+      this.forceVirtualizedListRowRerender();
+    }
```

Comment:
[very optional nit] just curious, do nesting `if`s here feel more readable to you? no worries at all if not, this is a very optional syntax comment
```suggestion
    if (isVirtualized) {
      if (
        prevProps.showIcons !== this.props.showIcons ||
        prevProps.paddingSize !== this.props.paddingSize ||
        prevProps.textWrap !== this.props.textWrap ||
        prevProps.onFocusBadge !== this.props.onFocusBadge ||
        prevProps.searchable !== this.props.searchable
      ) {
        this.forceVirtualizedListRowRerender();
      }
    }
```

Reply:
Personally additional if blocks make it more "floaty" and somewhat harder to read for me, but it's not really that bad that I would oppose to it. So if it's definitely better for you, I can update 🙂

Reply:
Updated in [this commit](https://github.com/elastic/eui/pull/7728/commits/08df104fdd9303374fad66eb2016268f0268e88e#diff-d472c9121e1553ab0cf2eeeaff574a59941a6f752955fdeedc8d055823501411R333) to separate into standalone conditions.

Reply:
Thanks!! Sorry for the slow response, I was going to say I have a very slight preference to nested but I'm also chill either way / leaving it as -is. In general it's just totally fascinating to me how other people read code vs how I read code. It's like when you wonder if the color blue is the same to you as it is to everyone else? 😆